src/lib.rs

```diff
@@ -251,4 +257,91 @@ mod test {
         let data: u16 = 0b1000_0000_0000_0001;
         assert!((single_ended_rdata_to_scaled_voltage(data as i16) - -V_MAX).abs() < EPS);
     }
+
+    const DEFAULT_CONFIG: u8 = 0b0000_0000;
+    const DEFAULT_STATUS: u8 = 0b0000_0001;
```

Comment:
maybe add a comment here denoting that the important thing here is that the MSB is not 1. Maybe it would be good to define this as something like:
```rust
const NOT_READY_STATUS: u8 = !0b1000_0000;
```